compiler/test/fail_compilation/issue11070.d

```diff
@@ -0,0 +1,17 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/issue11070.d(16): Error: undefined identifier `x`
+---
+*/
+
+int get() { return 1; }
+
+void test() {
+    import std.stdio : writeln;
+    switch (auto x = get()) {
+        default:
+            writeln("inside switch: ", x);
+    }
+    writeln("outside switch: ", x);
```

Comment:
```diff
-import std.stdio : writeln;
switch (auto x = get()) {
       default:
-             writeln("inside switch: ", x);
       }
-     writeln("outside switch: ", x);
+     x = 1;
```
please do not use `import std.anything` in the test suite

Reply:
Oh didnt know; thanks for the info!!